src/components/accueil/RightNavigation.tsx

```diff
@@ -118,6 +119,15 @@ export const RightNavigation = ({ activityUser }: { activityUser: User }) => {
           <Flag country={activityUser.country.isoCode}></Flag>
         </span>
       </div>
+      {isMediator && (
+        <Button
+          onClick={() => window.open(`https://prof.parlemonde.org/les-professeurs-partenaires/${activityUser.pseudo}/profile`, '_blank')}
+          style={{ overflow: 'hidden', margin: '2rem auto 2.5rem auto', display: 'block' }}
+          variant="outlined"
+        >
+          Voir la fiche du professeur
+        </Button>
```

Comment:
I would prefer to have directly a `<a>` link in the HTML, it's it purpose:
```suggestion
        <Button
          component="a"
          href={`https://prof.parlemonde.org/les-professeurs-partenaires/${activityUser.pseudo}/profile`}
          target="_blank"
          rel="noopener noreferrer"
          style={{ overflow: 'hidden', margin: '2rem auto 2.5rem auto', display: 'block' }}
          variant="outlined"
        >
          Voir la fiche du professeur
        </Button>
```